akka-docs/rst/java/logging.rst

```diff
@@ -123,6 +123,19 @@ If you want very detailed logging of all lifecycle changes of Actors (restarts,
       }
     }
 
+If you want unhandled messages logged at DEBUG (instead of being sent on the event bus as :class:`akka.actor.UnhandledMessage`):
```

Comment:
I don't think it is "instead". More like "in addition to".
In reality the UnhandledMessage is published to the event bus by the `unhandled` method in the Actor trait and if this is enabled a special listener picks it up and logs it.